lib/carrierwave/sequel.rb

```diff
@@ -11,6 +11,43 @@ def mount_uploader(column, uploader, options={}, &block)
       raise "You need to use Sequel 3.0 or higher. Please upgrade." unless ::Sequel::Model.respond_to?(:plugin)
       super
 
+      mod = Module.new
+      include mod
+      mod.class_eval <<-RUBY, __FILE__, __LINE__+1
+        def #{column}=(new_file)
+          if !(new_file.blank? && send(:#{column}).blank?)
+            modified!
+          end
+          super
+        end
+
+        def remove_#{column}!
```

Comment:
@mrbrdo Hm, strange... You're right, [ActiveRecord's version](https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/orm/activerecord.rb#L80) does modify model (in db) but original method doesn't ([1](https://github.com/carrierwaveuploader/carrierwave/blob/1e4e7b0bf20fee70d8ac3815d0921144f39e14cf/lib/carrierwave/mount.rb#L374), [2](https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/mounter.rb#L115), [3](https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/uploader/remove.rb#L13)). And I don't see callbacks (`before :remove`) in source code. What do you thank?